index.html

```diff
@@ -150,16 +130,12 @@
 
 fn main() {
     let mut server = Nickel::new();
-    
-    server.utilize(Nickel::json_body_parser());
 
-    fn post_handler (request: &Request, response: &mut Response) { 
-        let person = request.json_as::<&zwnj;Person>().unwrap();
-        let text = format!("Hello {} {}", person.firstname, person.lastname);
-        response.send(text.as_slice()); 
-    }
-    
-    server.post("/a/post/request", post_handler);
+    server.post("/a/post/request", middleware! { |request, response|
+        let person = request.json_as::&zwnj;Person>().unwrap();
```

Comment:
We need to use html entities for it to work but I guess it should actually be a &lt; instead of an &zwnj;